openmetrics/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# CHANGELOG - OpenMetrics
+
+## 1.0.0/ 2018-08-30
+
+* [FEATURE] adds openmetrics integration.
```

Comment:
You can remove the changelog entry, just leave the title `# CHANGELOG - OpenMetrics`.